lib/libvarnishapi/libvarnishapi.map

```diff
@@ -195,3 +195,11 @@ LIBVARNISHAPI_2.5 {	/* 2020-09-15 release */
     local:
 	*;
 };
+
+LIBVARNISHAPI_2.6 {	/* 2020-03-15 release */
```

Comment:
add a reminder to bump the API version for release?

Reply:
Let's have that chat offline.